### ISSUE TITLE
Feature request - Add 'footer' option to be equal header

### ISSUE THREAD

**Issue by murder** [dalance/procs#405]
This is for people like me who don't like to use pagers and sometimes create aliases that printf something after running a command/program, just to remember what each column corresponds to

This way, I can view what each column is in the end of the list as well.

```
 PID:▲  User             │ State Nice TTY    CPU  MEM   VmSize    VmRSS TCP    UDP      Read Write │ CPU Time Start            │ Command
                         │                   [%]  [%]  [bytes]  [bytes]                [B/s] [B/s] │                           │
 1      root             │ S        0        0.0  0.0 166.105M  10.145M                            │ 00:00:02 2022/12/21 09:34 │ /sbin/init splash
 2      root             │ S        0        0.0  0.0        0        0                            │ 00:00:00 2022/12/21 09:34 │ kthreadd
 3      root             │ I      -20        0.0  0.0        0        0                            │ 00:00:00 2022/12/21 09:34 │ rcu_gp
 4      root             │ I      -20        0.0  0.0        0        0                            │ 00:00:00 2022/12/21 09:34 │ rcu_par_gp
 5      root             │ I      -20        0.0  0.0        0        0                            │ 00:00:00 2022/12/21 09:34 │ slub_flushwq
 6      root             │ I      -20        0.0  0.0        0        0                            │ 00:00:00 2022/12/21 09:34 │ netns
 8      root             │ I      -20        0.0  0.0        0        0                            │ 00:00:00 2022/12/21 09:34 │ kworker/0:0H-kblockd
 10     root             │ I      -20        0.0  0.0        0        0                            │ 00:00:00 2022/12/21 09:34 │ mm_percpu_wq
 12     root             │ I        0        0.0  0.0        0        0                            │ 00:00:00 2022/12/21 09:34 │ rcu_tasks_kthread
 13     root             │ I        0        0.0  0.0        0        0                            │ 00:00:00 2022/12/21 09:34 │ rcu_tasks_rude_kthread
 14     root             │ I        0        0.0  0.0        0        0                            │ 00:00:00 2022/12/21 09:34 │ rcu_tasks_trace_kthread
 15     root             │ S        0        0.0  0.0        0        0                            │ 00:00:00 2022/12/21 09:34 │ ksoftirqd/0
 16     root             │ I        0        0.0  0.0        0        0                            │ 00:00:05 2022/12/21 09:34 │ rcu_preempt
 17     root             │ S        0        0.0  0.0        0        0                            │ 00:00:00 2022/12/21 09:34 │ rcub/1
 18     root             │ S        0        0.0  0.0        0        0                            │ 00:00:00 2022/12/21 09:34 │ migration/0
 19     root             │ S        0        0.0  0.0        0        0                            │ 00:00:00 2022/12/21 09:34 │ idle_inject/0
 21     root             │ S        0        0.0  0.0        0        0                            │ 00:00:00 2022/12/21 09:34 │ cpuhp/0
 22     root             │ S        0        0.0  0.0        0        0                            │ 00:00:00 2022/12/21 09:34 │ cpuhp/1
 23     root             │ S        0        0.0  0.0        0        0                            │ 00:00:00 2022/12/21 09:34 │ idle_inject/1
 24     root             │ S        0        0.0  0.0        0        0                            │ 00:00:00 2022/12/21 09:34 │ migration/1
 25     root             │ S        0        0.0  0.0        0        0                            │ 00:00:00 2022/12/21 09:34 │ ksoftirqd/1
 27     root             │ I      -20        0.0  0.0        0        0                            │ 00:00:00 2022/12/21 09:34 │ kworker/1:0H-events_highpri
 28     root             │ S        0        0.0  0.0        0        0                            │ 00:00:00 2022/12/21 09:34 │ cpuhp/2
 29     root             │ S        0        0.0  0.0        0        0                            │ 00:00:00 2022/12/21 09:34 │ idle_inject/2
 30     root             │ S        0        0.0  0.0        0        0                            │ 00:00:00 2022/12/21 09:34 │ migration/2
 31     root             │ S        0        0.0  0.0        0        0                            │ 00:00:00 2022/12/21 09:34 │ ksoftirqd/2
 33     root             │ I      -20        0.0  0.0        0        0                            │ 00:00:00 2022/12/21 09:34 │ kworker/2:0H-events_highpri
 34     root             │ S        0        0.0  0.0        0        0                            │ 00:00:00 2022/12/21 09:34 │ cpuhp/3
 35     root             │ S        0        0.0  0.0        0        0                            │ 00:00:00 2022/12/21 09:34 │ idle_inject/3
 36     root             │ S        0        0.0  0.0        0        0                            │ 00:00:00 2022/12/21 09:34 │ migration/3
 37     root             │ S        0        0.0  0.0        0        0                            │ 00:00:00 2022/12/21 09:34 │ ksoftirqd/3
 38     root             │ I        0        0.0  0.0        0        0                            │ 00:00:00 2022/12/21 09:34 │ kworker/3:0-mm_percpu_wq
 39     root             │ I      -20        0.0  0.0        0        0                            │ 00:00:00 2022/12/21 09:34 │ kworker/3:0H-events_highpri
 40     root             │ S        0        0.0  0.0        0        0                            │ 00:00:00 2022/12/21 09:34 │ cpuhp/4
 41     root             │ S        0        0.0  0.0        0        0                            │ 00:00:00 2022/12/21 09:34 │ idle_inject/4
 42     root             │ S        0        0.0  0.0        0        0                            │ 00:00:01 2022/12/21 09:34 │ migration/4
 43     root             │ S        0        0.0  0.0        0        0                            │ 00:00:00 2022/12/21 09:34 │ ksoftirqd/4
 45     root             │ I      -20        0.0  0.0        0        0                            │ 00:00:00 2022/12/21 09:34 │ kworker/4:0H-events_highpri
 46     root             │ S        0        0.0  0.0        0        0                            │ 00:00:00 2022/12/21 09:34 │ cpuhp/5
 47     root             │ S        0        0.0  0.0        0        0                            │ 00:00:00 2022/12/21 09:34 │ idle_inject/5
 48     root             │ S        0        0.0  0.0        0        0                            │ 00:00:00 2022/12/21 09:34 │ migration/5
 49     root             │ S        0        0.0  0.0        0        0                            │ 00:00:00 2022/12/21 09:34 │ ksoftirqd/5
 51     root             │ I      -20        0.0  0.0        0        0                            │ 00:00:00 2022/12/21 09:34 │ kworker/5:0H-events_highpri
 52     root             │ S        0        0.0  0.0        0        0                            │ 00:00:00 2022/12/21 09:34 │ cpuhp/6
 53     root             │ S        0        0.0  0.0        0        0                            │ 00:00:00 2022/12/21 09:34 │ idle_inject/6
 54     root             │ S        0        0.0  0.0        0        0                            │ 00:00:01 2022/12/21 09:34 │ migration/6
 55     root             │ S        0        0.0  0.0        0        0                            │ 00:00:00 2022/12/21 09:34 │ ksoftirqd/6
 57     root             │ I      -20        0.0  0.0        0        0                            │ 00:00:00 2022/12/21 09:34 │ kworker/6:0H-events_highpri
 58     root             │ S        0        0.0  0.0        0        0                            │ 00:00:00 2022/12/21 09:34 │ cpuhp/7
 59     root             │ S        0        0.0  0.0        0        0                            │ 00:00:00 2022/12/21 09:34 │ idle_inject/7
 60     root             │ S        0        0.0  0.0        0        0                            │ 00:00:00 2022/12/21 09:34 │ migration/7
 61     root             │ S        0        0.0  0.0        0        0                            │ 00:00:00 2022/12/21 09:34 │ ksoftirqd/7
 63     root             │ I      -20        0.0  0.0        0        0                            │ 00:00:00 2022/12/21 09:34 │ kworker/7:0H-events_highpri
 64     root             │ S        0        0.0  0.0        0        0                            │ 00:00:00 2022/12/21 09:34 │ cpuhp/8
 65     root             │ S        0        0.0  0.0        0        0                            │ 00:00:00 2022/12/21 09:34 │ idle_inject/8
 66     root             │ S        0        0.0  0.0        0        0                            │ 00:00:00 2022/12/21 09:34 │ migration/8
 67     root             │ S        0        0.0  0.0        0        0                            │ 00:00:00 2022/12/21 09:34 │ ksoftirqd/8
 69     root             │ I      -20        0.0  0.0        0        0                            │ 00:00:00 2022/12/21 09:34 │ kworker/8:0H-events_highpri
 70     root             │ S        0        0.0  0.0        0        0                            │ 00:00:00 2022/12/21 09:34 │ cpuhp/9
 71     root             │ S        0        0.0  0.0        0        0                            │ 00:00:00 2022/12/21 09:34 │ idle_inject/9
 72     root             │ S        0        0.0  0.0        0        0                            │ 00:00:00 2022/12/21 09:34 │ migration/9
 73     root             │ S        0        0.0  0.0        0        0                            │ 00:00:05 2022/12/21 09:34 │ ksoftirqd/9
 75     root             │ I      -20        0.0  0.0        0        0                            │ 00:00:00 2022/12/21 09:34 │ kworker/9:0H-events_highpri
 76     root             │ S        0        0.0  0.0        0        0                            │ 00:00:00 2022/12/21 09:34 │ cpuhp/10
 77     root             │ S        0        0.0  0.0        0        0                            │ 00:00:00 2022/12/21 09:34 │ idle_inject/10
 78     root             │ S        0        0.0  0.0        0        0                            │ 00:00:01 2022/12/21 09:34 │ migration/10
 79     root             │ S        0        0.0  0.0        0        0                            │ 00:00:00 2022/12/21 09:34 │ ksoftirqd/10
 81     root             │ I      -20        0.0  0.0        0        0                            │ 00:00:00 2022/12/21 09:34 │ kworker/10:0H-events_highpri
 82     root             │ S        0        0.0  0.0        0        0                            │ 00:00:00 2022/12/21 09:34 │ rcub/2
 83     root             │ S        0        0.0  0.0        0        0                            │ 00:00:00 2022/12/21 09:34 │ cpuhp/11
 84     root             │ S        0        0.0  0.0        0        0                            │ 00:00:00 2022/12/21 09:34 │ idle_inject/11
 85     root             │ S        0        0.0  0.0        0        0                            │ 00:00:00 2022/12/21 09:34 │ migration/11
 86     root             │ S        0        0.0  0.0        0        0                            │ 00:00:00 2022/12/21 09:34 │ ksoftirqd/11
 88     root             │ I      -20        0.0  0.0        0        0                            │ 00:00:00 2022/12/21 09:34 │ kworker/11:0H-events_highpri
 89     root             │ S        0        0.0  0.0        0        0                            │ 00:00:00 2022/12/21 09:34 │ cpuhp/12
 90     root             │ S        0        0.0  0.0        0        0                            │ 00:00:00 2022/12/21 09:34 │ idle_inject/12
 91     root             │ S        0        0.0  0.0        0        0                            │ 00:00:00 2022/12/21 09:34 │ migration/12
 92     root             │ S        0        0.0  0.0        0        0                            │ 00:00:00 2022/12/21 09:34 │ ksoftirqd/12
 94     root             │ I      -20        0.0  0.0        0        0                            │ 00:00:00 2022/12/21 09:34 │ kworker/12:0H-events_highpri
 95     root             │ S        0        0.0  0.0        0        0                            │ 00:00:00 2022/12/21 09:34 │ cpuhp/13
 96     root             │ S        0        0.0  0.0        0        0                            │ 00:00:00 2022/12/21 09:34 │ idle_inject/13
 97     root             │ S        0        0.0  0.0        0        0                            │ 00:00:00 2022/12/21 09:34 │ migration/13
 98     root             │ S        0        0.0  0.0        0        0                            │ 00:00:00 2022/12/21 09:34 │ ksoftirqd/13
 100    root             │ I      -20        0.0  0.0        0        0                            │ 00:00:00 2022/12/21 09:34 │ kworker/13:0H-events_highpri
 101    root             │ S        0        0.0  0.0        0        0                            │ 00:00:00 2022/12/21 09:34 │ cpuhp/14
 102    root             │ S        0        0.0  0.0        0        0                            │ 00:00:00 2022/12/21 09:34 │ idle_inject/14
 103    root             │ S        0        0.0  0.0        0        0                            │ 00:00:01 2022/12/21 09:34 │ migration/14
 104    root             │ S        0        0.0  0.0        0        0                            │ 00:00:00 2022/12/21 09:34 │ ksoftirqd/14
 106    root             │ I      -20        0.0  0.0        0        0                            │ 00:00:00 2022/12/21 09:34 │ kworker/14:0H-events_highpri
 107    root             │ S        0        0.0  0.0        0        0                            │ 00:00:00 2022/12/21 09:34 │ cpuhp/15
 108    root             │ S        0        0.0  0.0        0        0                            │ 00:00:00 2022/12/21 09:34 │ idle_inject/15
 109    root             │ S        0        0.0  0.0        0        0                            │ 00:00:00 2022/12/21 09:34 │ migration/15
 110    root             │ S        0        0.0  0.0        0        0                            │ 00:00:00 2022/12/21 09:34 │ ksoftirqd/15
 111    root             │ I        0        0.0  0.0        0        0                            │ 00:00:00 2022/12/21 09:34 │ kworker/15:0-mm_percpu_wq
 112    root             │ I      -20        0.0  0.0        0        0                            │ 00:00:00 2022/12/21 09:34 │ kworker/15:0H-events_highpri
 113    root             │ S        0        0.0  0.0        0        0                            │ 00:00:00 2022/12/21 09:34 │ cpuhp/16
 114    root             │ S        0        0.0  0.0        0        0                            │ 00:00:00 2022/12/21 09:34 │ idle_inject/16
 115    root             │ S        0        0.0  0.0        0        0                            │ 00:00:04 2022/12/21 09:34 │ migration/16
 116    root             │ S        0        0.0  0.0        0        0                            │ 00:00:00 2022/12/21 09:34 │ ksoftirqd/16
 118    root             │ I      -20        0.0  0.0        0        0                            │ 00:00:00 2022/12/21 09:34 │ kworker/16:0H-events_highpri
 119    root             │ S        0        0.0  0.0        0        0                            │ 00:00:00 2022/12/21 09:34 │ cpuhp/17
 120    root             │ S        0        0.0  0.0        0        0                            │ 00:00:00 2022/12/21 09:34 │ idle_inject/17
 121    root             │ S        0        0.0  0.0        0        0                            │ 00:00:03 2022/12/21 09:34 │ migration/17
                         │                   [%]  [%]  [bytes]  [bytes]                [B/s] [B/s] │                           │
 PID:▲  User             │ State Nice TTY    CPU  MEM   VmSize    VmRSS TCP    UDP      Read Write │ CPU Time Start            │ Command
                         │                   [%]  [%]  [bytes]  [bytes]                [B/s] [B/s] │                           │
```

**Comment by dalance** [comment]
I added `show_footer` option.
https://github.com/dalance/procs#display-section

**Comment by dalance** [comment]
I released v0.14.0.
